app/renderer/js/main.js

```diff
@@ -87,7 +90,12 @@ class ServerManagerView {
 			showNotification: true,
 			betaUpdate: false,
 			silent: false,
-			lastActiveTab: 0
+			lastActiveTab: 0,
+			dnd: false,
+			oldSettings: {
+				showNotification: ConfigUtil.getConfigItem('showNotification') || true,
```

Comment:
@cPhost @akashnimare This statement gives true always. So, we should go for conditional statement.

```js
showNotification: ConfigUtil.getConfigItem('showNotification') === null ? true : ConfigUtil.getConfigItem('showNotification'),
```

Reply:
It shouldn't return true if its disabled in settings. It returns `true` since the default it true.

Reply:
To be clear this doesn't need a check!

Reply:
Err! The way it is is fine.